features/server-sent-events.yml

```
@@ -0,0 +1,17 @@
+name: Server-sent events
+description: "The `EventSource` API creates a persistent one-way connection to the originating server for receiving streams of events."
```

Continuing from https://github.com/web-platform-dx/web-features/pull/1913#discussion_r1792707456

```suggestion
description: The `EventSource` API creates a connection to a server and listens to a stream of events sent by the server.
```